### PR TITLE
remove `@Doron-Bargo` from logzio exporter `CODEOWNERS`

### PR DESCRIPTION
**Pull request by yotamloe**
**Description:** 
Remove @Doron-Bargo  from logzio exporter `CODEOWNERS` as requested

**Link to tracking Issue:** 
#29719

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -289,4 +289,3 @@ reports/distributions/liatrio.yaml  @open-telemetry/collector-contrib-approvers
 
 
 ## UNMAINTAINED components
-
```
Comment (songy23) [inline .github/CODEOWNERS:292]: New line

**Review by songy23 — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
Thanks @yotamloe! An update to the component's README.md and [metadata.yaml](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/logzioexporter/metadata.yaml)  will also be needed

File: .github/CODEOWNERS (hunk 1 above)
Comment (bogdandrutu) [reply .github/CODEOWNERS:292]: ping

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -288,5 +288,4 @@ reports/distributions/sumo.yaml     @open-telemetry/collector-contrib-approvers
 reports/distributions/liatrio.yaml  @open-telemetry/collector-contrib-approvers @adrielp
 
 
-## UNMAINTAINED components
-
+## UNMAINTAINED components
```
Comment (codeboten) [inline .github/CODEOWNERS:291]:
```suggestion
## UNMAINTAINED components

```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -289,4 +289,3 @@ reports/distributions/liatrio.yaml  @open-telemetry/collector-contrib-approvers
 
 
 ## UNMAINTAINED components
```
Comment (codeboten) [inline .github/CODEOWNERS:291]:
```suggestion
## UNMAINTAINED components

```

**Review by codeboten — APPROVED** [review]

**Pull request merged**